sparse_strips/vello_bench/src/flatten.rs

```diff
@@ -68,7 +69,9 @@ pub fn strokes(c: &mut Criterion) {
                             width: path.stroke_width as f64,
                             ..Default::default()
                         };
-                        paths.push(flatten::expand_stroke(path.path.iter(), &stroke, 0.25));
+                        let mut stroke_ctx = StrokeCtx::default();
```

Comment:
Do we not want to move the creation of `StrokeCtx` outside `b.iter` to allow its reuse?

```rs
                let mut stroke_ctx = StrokeCtx::default();
                b.iter(|| {
                    let mut paths = vec![];

                    for path in &$item.strokes {
                        let stroke = Stroke {
                            width: path.stroke_width as f64,
                            ..Default::default()
                        };
                        flatten::expand_stroke(path.path.iter(), &stroke, 0.25, &mut stroke_ctx);
                        paths.push(stroke_ctx.output().clone());
                    }

                    std::hint::black_box(&paths);
                })
```